test/providers/cloudflare_test.rb

```diff
@@ -160,6 +161,24 @@ def test_build_srv_from_api
     assert_equal(3600, record.ttl)
   end
 
+  def test_build_alias_from_api
```

Comment:
Can you also add a test for the `build_api_body` ? Just to make sure something fails if anyone changes the implementation...